app/models/school.rb

```diff
@@ -25,6 +25,7 @@
 #
 class School < ApplicationRecord
   include PgSearch::Model
+  include PlacementQuery
```

Comment:
can you mix in `PlacementQuery` here? i think it relies on the presence of `trainees`, which you don't have in `School`